libraries/AP_Baro/AP_Baro.cpp

```diff
@@ -197,6 +229,10 @@ void AP_Baro::update_calibration()
             _EAS2TAS = 0;
         }
     }
+
+    // update and save base pressure (persistent between boots) with primary baro ground calibration (not persistent)
+    _base_pressure.set_and_save(get_ground_pressure());
+    _base_pressure.notify();
```

Comment:
No. This was introduced with a button in QGC to 'zero depth' (which uses mav_cmd_preflight_calibration), effectively replacing the need to have the RESET parameter, which was included much earlier on.